torch/csrc/api/include/torch/nn/modules/embedding.h

```diff
@@ -12,20 +12,32 @@ namespace nn {
 
 /// Options for the `Embedding` module.
 struct TORCH_API EmbeddingOptions {
-  EmbeddingOptions(int64_t count, int64_t dimension);
-  /// The number of embeddings (number of rows in the table).
-  TORCH_ARG(int64_t, count);
-  /// The size of each embedding vector (number of columns in the table).
-  TORCH_ARG(int64_t, dimension);
+  EmbeddingOptions(int64_t num_embeddings, int64_t embedding_dim);
+  // The number of embeddings (number of rows in the table).
+  TORCH_ARG(int64_t, num_embeddings);
+  // The size of each embedding vector (number of columns in the table).
+  TORCH_ARG(int64_t, embedding_dim);
+  // If given, pads the output with the embedding vector at :attr:`padding_idx (initialized to zeros) whenever it encounters the index.
+  TORCH_ARG(c10::optional<int64_t>, padding_idx)=c10::nullopt;
+  // If given, each embedding vector with norm larger than :attr:`max_norm` is renormalized to have norm :attr:`max_norm`.
+  TORCH_ARG(c10::optional<float>, max_norm)=c10::nullopt;
+  // The p of the p-norm to compute for the :attr:`max_norm` option. Default ``2``
+  TORCH_ARG(float, norm_type)=2.;
+  // If given, this will scale gradients by the inverse of frequency of the words in the mini-batch. Default ``False``.
+  TORCH_ARG(bool, scale_grad_by_freq)=false;
+  // If ``True``, gradient w.r.t. :attr:`weight` matrix will be a sparse tensor.
+  TORCH_ARG(bool, sparse)=false;
+  // The learnable weights of the module of shape (num_embeddings, embedding_dim) initialized from a normal distribution with mean=0, std=1.
```

Comment:
We probably shouldn't impose how the user should initialize `_weight`, and we should say:
```suggestion
  // The learnable weights of the module of shape (num_embeddings, embedding_dim).
```